net/src/packet/meta.rs

```diff
@@ -77,10 +77,15 @@ pub struct PacketMeta {
     pub done: Option<DoneReason>, /* if Some, the reason why a packet was marked as done, including delivery to NF */
     pub src_vni: Option<Vni>, /* the vni value of a received vxlan encap packet, if destined to gateway */
     pub dst_vni: Option<Vni>, /* the vni value of a vxlan packet re-encapsulated by the gateway */
-
-    #[cfg(test)]
-    /* Keep the Packet in spite of calling packet.enforce(). This is for testing */
-    pub keep: bool,
+    pub keep: bool,           /* Keep the Packet even if it should be dropped */
```

Comment:
In the future, should we move this to dont_keep or something so that keep is the default behavior?

Reply:
Can we create an issue to detect kept packets that are inadvertently destroyed?

Reply:
> In the future, should we move this to dont_keep or something so that keep is the default behavior?
We can. Our "initial" stages / NFs called filter_map() when iterating over packets because the assumption was that they would drop packets. If we want to go for an approach where that should not happen, we could replace all filter_maps so that they would never filter. That being said, I think that the current approach where they may filter and we can override the behavior of packet.enforce() by setting the keep flag is the most flexible at the moment.

Reply:
> Can we create an issue to detect kept packets that are inadvertently destroyed?